assets/js/base/components/cart-checkout/totals/totals-coupon-code-input/index.js

```diff
@@ -39,11 +39,11 @@ const TotalsCouponCodeInput = ( {
 		}
 	}, [ isLoading, couponValue, validationError ] );
 
-	const textInputId = `wc-block-coupon-code__input-${ instanceId }`;
+	const textInputId = `wc-block-components-totals-coupon__input-${ instanceId }`;
 
 	return (
 		<Panel
-			className="wc-block-coupon-code"
+			className="wc-block-components-totals-coupon"
```

Comment:
I created a follow-up issue to rename the subdirectories of `base/components/cart-checkout/totals/*` to simplify them and match these new class names: #2712.